### PR TITLE
Standardise CLI param placeholders.

### LINKED ISSUE #182: Make placeholders for user input (passwords, etc) consistant
We are completely inconsistent in how we should that something in an example needs to have a user specified value.  I have seen (and done) things like:
```
dcrctl -u USER -p PASS getblockcount
```
```
dcrctl -u USER -p sekret getblockcount
```
```
dcrctl -u 'USER' -p 'PASS' getblockcount
```
```
dcrctl -u <USER> -p <PASS> getblockcount
```

We need to pick one and update all the docs to use it.  Suggestions are welcome.  I hate the use of quotes to indicated it (I've seen that trip up users too many times) so that is off the table.  I'd prefer `<USER>` myself but could be convinced of others.

### PR DESCRIPTION
**Pull request by jholdstock**
Closes #182

### CODE REVIEW

File: docs/getting-started/startup-basics.md [hunk 1]
```diff
@@ -53,14 +53,14 @@ At the very minimum, for `dcrd`, `dcrwallet`, and `dcrctl` to be able to communi
 4. Create an empty text file and save it as `dcrctl.conf` in `dcrctl`'s home directory.
 5. Choose an arbitrary username and password, these will only be used for each application to communicate via remote procedure call. The easiest configuration is to set them all equal.
 6. Inside `dcrd.conf`, underneath `[Application Options]`, add the following lines:<br /><br />
-        rpcuser=chosen-username<br />
-        rpcpass=chosen-password<br /><br />
+        rpcuser=<chosen-username><br />
```
Comment (gratefulcheddar) [inline docs/getting-started/startup-basics.md]: Put these lines in backquotes (`) otherwise Mkdocs tries to render the brackets as HTML tags.

**Comment by jholdstock** [comment]
Done and I also found one more place to add the <> brackets.

File: docs/advanced/program-options.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ You can always get a list of all options by using the `-h` option.
 ## **Usage**
 
 ```
-dcrd [OPTIONS]
+dcrd <OPTIONS>
```
Comment (jcvernaleo) [inline docs/advanced/program-options.md]: Actually, I think the square brackets were correct there.  This is not a single value that the user replaces with their own stuff but a whole bunch of stuff.  Much like the way most man pages do it:
```
SYNOPSIS
       ls [OPTION]... [FILE]...
```

**Comment by jcvernaleo** [comment]
All good now.  Thanks!

**Pull request merged**